### PR TITLE
[Feature] Last active at tracking

### LINKED ISSUE #11426: ✨ Track last sign-in for all users
## ✨ Feature

We want to know the number of active users on the site. The measure we intend to use for this is whether or not they have signed in to the site. So let's start collecting that!

## 🕵️ Details

Add a field that stores the timestamp of someone successfully signing into the site for all users.

### PR DESCRIPTION
**Pull request by vd1992**
🤖 Resolves #11426 

## 👋 Introduction

Adds a field to track last active and handles updating it through an authorization callback

## 🧪 Testing

1. Migration works
2. Open up database tool and check logging in updates the new field

## 📸 Screenshot

image

### CODE REVIEW

File: api/app/Providers/AuthServiceProvider.php
Comment (petertgiles) [inline api/app/Providers/AuthServiceProvider.php:1]: Hmm, I'm not sure this is the best place for the timestamp update.  This gets called for every request so the timestamp is continually being updated - not just when they log in.  I recommend moving this over to the auth callback so it only occurs on login.  You can see that the token that we receive and decode there does have the sub available, just like we grab the nonce.
image

**Review by petertgiles — CHANGES_REQUESTED** [review]
Looking good!  I suggested a change to the implementation to get a little closer to the AC.

File: api/app/Providers/AuthServiceProvider.php
Comment (vd1992) [reply api/app/Providers/AuthServiceProvider.php:1]: Oh, there actually is something called auth callback specifically and it wasn't meant in the general sense 😊 
Tweaked in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11437/commits/91130bdc427a6eddcc0a988b1cf990c7012849f1
Wasn't sure how to test it

**Review by petertgiles — COMMENTED** [review]
Ooh, a bit stuck now...

File: api/app/Providers/AuthServiceProvider.php
Comment (petertgiles) [reply api/app/Providers/AuthServiceProvider.php:1]: OK, decision in standup is that this is good enough for now.  I'll create a new issue to adjust this.

**Pull request merged**